### PR TITLE
feat: nft viewer background

### PR DESCRIPTION
**Pull request by mderrick**
### 📝 Description

NFT Viewer slight redesign
- Move floor price
- Change currency icon to the circle so it is more legible
- Added fixed and blurred background which fades out on scroll
- Made header transparent with a title and background appearing on scroll

### ❓ Context

- **Impacted projects**: `ledger-mobile` 
- **Linked resource(s)**: `LIVE-7690` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 📸 Demo

https://github.com/LedgerHQ/ledger-live/assets/1044686/dd3c83ac-711e-4c74-a3aa-cec8eb15bbd8

https://github.com/LedgerHQ/ledger-live/assets/1044686/724642de-d5a5-45b9-ba60-654184e43643

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx [hunk 1]
```diff
@@ -159,13 +161,12 @@ const NftViewer = ({ route }: Props) => {
...
   };
   const { t } = useTranslation();
-  const navigation =
-    useNavigation<
-      CompositeNavigationProp<
-        StackNavigatorNavigation<NftNavigatorParamList, ScreenName.NftViewer>,
-        StackNavigatorNavigation<BaseNavigatorStackParamList>
-      >
-    >();
+  const navigation = useNavigation<
```
Comment (mderrick) [inline apps/ledger-live-mobile/src/components/Nft/NftViewer.tsx:164]: Nothing changed here, prettier was not running in this file so it reformatted other things when I got it to run.

File: apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+const BLUR_RADIUS = 45;
+
+interface Props {
+  readonly src: string;
+  readonly scrollY: Animated.Value;
+}
+
+const NftViewerBackground: FC<Props> = ({ src, scrollY }) => {
+  const { dark } = useTheme();
+  const gradientColor = dark ? "#131214" : "#fff";
```
Comment (mderrick) [inline apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx:17]: `#131214` doesn't exist in the theme colors. I also see `useTheme` imported from `react-native` and styled components in various places. What's the consensus here for doing this kind of thing?

Comment (juan-cortes) [reply apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx:17]: I believe `#131214` is `background.main` for dark and `#ffffff` for light.

File: apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  const overlayColor = dark ? "#000" : "#fff";
```
Comment (juan-cortes) [inline apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx:18]: Probably the same here, there must be a palette color that either it or close to it. If there's not a matching color you should ping design for them to either add a new one (dont do it without asking to not pollute the palette) or choose an existing one.

File: apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  const opacity = scrollY.interpolate({
+    inputRange: [0, OPACITY_ANIMATION_PERIOD],
+    outputRange: [1, 0],
+    extrapolateRight: "clamp",
+  });
+  return (
+    <Animated.View style={[styles.root]}>
+      {/* NOTE: We have to aniamte the image only, iOs works fine animating out the entire
```
Comment (juan-cortes) [inline apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx:26]: typo "aniamte" "iOs"

File: apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+      view but Android doesn't play nice with the gradient and the text in the content
+      becomes ineligible for a brief time when scrolling */}
```
Comment (juan-cortes) [inline apps/ledger-live-mobile/src/components/Nft/NftViewerBackground.tsx:28]: ineligible sounds like a false friend, do you mean "unreadable"?

**Review by juan-cortes — CHANGES_REQUESTED** [review]

**Review by juan-cortes — APPROVED** [review]
lgtm

**Pull request merged**